src/h5web/BreadcrumbsBar.tsx

```diff
@@ -40,7 +40,7 @@ function BreadcrumbsBar(props: Props): ReactElement {
           {selectedNode?.parents.slice(firstParentIndex).map((member) => (
             <Fragment key={(member as HDF5HardLink).id}>
               <span className={styles.crumb}>{member.title}</span>
-              <FiChevronRight className={styles.separator} />
+              <FiChevronRight className={styles.separator} title="/" />
```

Comment:
What follows are tweaks that help me query elements in an accessible manner in Cypress.

Reply:
Here, this allows the h1 to have a `textContent` of `crumb1 / crumb2 / crumb3` instead of `crumb1crumb2crumb3`.